### DISCUSSION TITLE
100% difference on same pdf

### DISCUSSION THREAD

**Discussion by e-morse** [red6/pdfcompare#118] [Q&A]
I have several types of generated pdfs that I'm verifying using this library. The font rendering for these pdfs is different in different os, so I have tolerance set to 2%. This works as intended for every type of pdf except 1. It shows 100% difference on this type of pdf, even though when I look at it in the UI, there are only minor font differences. Is there a case where this might happen? How can I resolve this?

**Comment by finsterwalder** [comment]
It's really hard to say without inspecting the PDFs.
Usually it hat something to do with how the PDFs are rendered by the pdfbox library that I use and I don't really have any influence on that.
Some very general hints: Take a look at the difference PDF and see what is marked as differing.

**Reply by e-morse** [reply]
That's what's weird. nothing is marked as different. There's a green indicator of differences across the top and down the side, but no differences marked in the document.

**Reply by e-morse** [reply]
For anyone wondering, the problem turned out to be the file name i inputted. It was looking for 1234_Single.pdf, but it needed to be looking for 1234_single.pdf.  🤷

**Reply by finsterwalder** [reply]
:+1: 
When the side of the page is marked in green or red completely, that means, that the page is not at all in the expected result or that it's missing in the original version, but present in the expected result.
